WalletWasabi/Blockchain/Keys/KeyManager.cs

```diff
@@ -418,7 +418,7 @@ public IEnumerable<byte[]> GetPubKeyScriptBytes()
 			}
 		}
 
-		public HdPubKey GetKeyForScriptPubKey(Script scriptPubKey)
+		public HdPubKey? GetKeyForScriptPubKey(Script scriptPubKey)
```

Comment:
Please change this to use the `TryGet` pattern.

Reply:
Done